tests/unit_test/generic_web_api/generic_web_api_config_test.py

```diff
@@ -232,24 +231,6 @@ def test_should_read_batch_size(self):
         })
         assert web_api_config.batch_size == 123
 
-    def test_should_read_request_builder_name_and_parameters(self):
```

Comment:
I think this test wasn't `civi` specific in so far that it tested request builder specific parameters. Although it was `civi` specific in that it's was only request builder that used them.

So we should probably also remove `request_builder_parameters` from `WebApiDynamicRequestBuilder` (and the corresponding support for it in the config).